config/default_metrics.py

```diff
@@ -0,0 +1,27 @@
+from lsst.ap.verify.measurements.profiling import TimingMetricConfig
+
+config.jobFileTemplate = "ap_verify.metricTask{id}.{dataId}.verify.json"
+
+config.measurers = ["timing"]
+
+timingConfigs = {
+    "apPipe.runDataRef": "ap_pipe.ApPipeTime",
+    "apPipe:ccdProcessor.runDataRef": "pipe_tasks.ProcessCcdTime",
+    "apPipe:ccdProcessor:isr.runDataRef": "ip_isr.IsrTime",
+    "apPipe:ccdProcessor:charImage.runDataRef": "pipe_tasks.CharacterizeImageTime",
+    "apPipe:ccdProcessor:calibrate.runDataRef": "pipe_tasks.CalibrateTime",
+    "apPipe:differencer.runDataRef": "pipe_tasks.ImageDifferenceTime",
+    "apPipe:differencer:astrometer.loadAndMatch": "meas_astrom.AstrometryTime",
+    "apPipe:differencer:register.run": "pipe_tasks.RegisterImageTime",
+    "apPipe:differencer:subtract.subtractExposures": "ip_diffim.ImagePsfMatchTime",
+    "apPipe:differencer:detection.run": "meas_algorithms.SourceDetectionTime",
+    "apPipe:differencer:measurement.run": "ip_diffim.DipoleFitTime",
+    "apPipe:associator.run": "ap_association.AssociationTime",
```

Comment:
It seems unfortunate that the names had to be lifted out of the yaml and stringified: makes it a bit harder to scan. But probably inevitable.

Reply:
I think this is a usability improvement overall, since "what tasks do we want to run?" is something Stack users expect to live in a Python config file. The only real problem is that for verification we want to run a *lot* of tasks.

I've discussed the "lots of tasks" aspect with @TallJimbo and he said the Gen 3 framework will eventually support grouping pipeline tasks in some way. Hopefully that will, if not make config files simpler, at least let us segregate the parts that are and are not likely to change.